_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -0,0 +1,295 @@
+---
+id: qt1vmo
+name: Image accessible name is descriptive
+rule_type: atomic
+description: |
+  This rule checks that images with an accessible name describe the topic or purpose.
```

Comment:
I think this is grammatically incorrect.
The subject of "describe" is here "images with an accessible name" and not just "accessible name". But the rule checks that the accessible name describes the purpose of the image

Reply:
English is wonderfully nuanced and easily taken the wrong way. I would suggest something more like 
```suggestion
  This rule checks that images with an accessible name describe their topic or purpose.
```

Reply:
"the topic or purpose" could refer to something else on the page, not necessarily the subject of the sentence or the rule. English is wonderful for misinterpretation. "their topic or purpose" is less likely to be misinterpreted.